apps/condo/migrations/20210707112100-00027-insert_default_organization_employee_roles.js

```diff
@@ -0,0 +1,141 @@
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+    
+    -- Dispatcher
+    INSERT INTO "OrganizationEmployeeRole" (
+        id, dv, v, sender, organization, "createdAt", "updatedAt",
+        name,
+        "canManageOrganization",
+        "canManageEmployees",
+        "canManageRoles",
+        "canManageIntegrations",
+        "canManageProperties",
+        "canManageTickets",
+        "canManageContacts",
+        "canManageTicketComments"
+    )
+    SELECT
+        gen_random_uuid(), 1, v, '{"dv": 1, "fingerprint": "migration", "timestamp": "20210707112100"}'::json, org.id, now(), now(),
+        case when org.country = 'ru' then 'Диспетчер'
+             when org.country = 'en' then 'Dispatcher'
+        end,
+        false,
+        false,
+        false,
+        false,
+        true,
+        true,
+        true,
+        true
+    FROM "Organization" AS org
+    WHERE NOT EXISTS(SELECT name, organization FROM "OrganizationEmployeeRole" WHERE organization = org.id AND (name = 'Диспетчер' OR name = 'Dispatcher'));
```

Comment:
Select list is redundant here:

```suggestion
    WHERE NOT EXISTS(SELECT FROM "OrganizationEmployeeRole" WHERE organization = org.id AND (name = 'Диспетчер' OR name = 'Dispatcher'));
```

Reply:
fixed